crates/nargo_cli/tests/test_data_ssa_refactor/brillig_oracle/src/main.nr

```diff
@@ -21,3 +27,22 @@ unconstrained fn oracle_print_array_wrapper(arr: [Field; 2]) {
     oracle_print_array(arr);
 }
 
+#[oracle(get_number_sequence)]
+unconstrained fn get_number_sequence(_size: Field) -> [Field] {}
+
+#[oracle(get_reverse_number_sequence)]
+unconstrained fn get_reverse_number_sequence(_size: Field) -> [Field] {}
```

Comment:
Why do these need to be oracle functions?

```rs
// This is usually called `iota`
unconstrained fn get_number_sequence(length: Field) -> [Field] {
    let mut seq: [Field] = [];
    for i in 0 .. length {
        seq = seq.push_back(i);
    }
    seq
}
```

Reply:
They don't need to be. I was more testing returning an array of unknown size from an oracle. I can make it something more useful but it was really just for testing purposes

Reply:
If this is for testing lets make a note of it and create an issue to remove them later since they seem unneeded.